typescript/src/api-doc-types/ConstExportDoc.ts

```diff
@@ -15,11 +15,15 @@ export class ConstExportDoc extends ExportDoc {
     super(host, moduleDoc, symbol, symbol.valueDeclaration!, aliasSymbol);
   }
 
-  private getTypeString() {
-    if (this.variableDeclaration.type) {
-      return this.typeChecker.typeToString(this.typeChecker.getTypeFromTypeNode(this.variableDeclaration.type));
-    } else if (this.variableDeclaration.initializer) {
-      return this.typeChecker.typeToString(this.typeChecker.getTypeAtLocation(this.variableDeclaration.initializer));
+  private getTypeString(): string | undefined {
+    if (!this.variableDeclaration.type && !this.variableDeclaration.initializer) {
+      return undefined;
     }
+
+    const typeNode = this.variableDeclaration.type ?
```

Comment:
Strictly this is a `type` not a `typeNode` :-)

Reply:
Ahh true, previously the ternary just referred to the `TypeNode` from the variable declaration. I will update it.

Reply:
Nevermind 😄 so should we just keep it like that?

Reply:
Now worries - happy to get it in as-is. If I get round to doing a major refactoring later, it would probably change again anyway.

Reply:
Sounds good.